### ISSUE TITLE
Gravity Forms field borders have insufficient color contrast

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/justgetsimple#54]
Per recommendation from Ablr: "the text boxes, radio buttons, and checkboxes outline is light gray (#C5C5CC) and has a lower than 3 to 1 contrast with surrounding."

https://www.w3.org/TR/WCAG21/#non-text-contrast

Recommend increasing the color contrast in the theme.

**Comment by isabisa** [comment]
Hey @andrewmarino just checking... this feedback was specifically about our website, not Simple. Is there a reason the issues are logged in this repo and not on our website? (Besides a possible brain fart?)

**Comment by andrewmarino** [comment]
Correct. We haven't started on the Unity Web Agency fixes yet but it's the same issue on Simple's base form styles as well.
